<launch>
    <arg name="rviz" default="true" />

    <!-- start the Realsense -->
    <include file="$(find realsense2_viewer)/launch/realsense2_viewer.launch" />

    <!-- start the rviz -->
    <node if="$(arg rviz)" name="$(anon rviz)" pkg="rviz" type="rviz" respawn="false" args="-d $(find oil_filler_pose)/launch/oil_filler_detect.rviz" output="screen" />

    <!-- start the oil_filler_pose -->
	<!-- <node name="oil_filler_pose" pkg="oil_filler_pose" type="oil_filler_pose_realsense" respawn="true" output="screen">

       <param name="camera_frame" value="camera_color_optical_frame" />

       <param name="show" value="true" />

	</node> -->

</launch>
